lib/u3d/utils.rb

```diff
@@ -151,6 +151,12 @@ def ensure_dir(dir)
         FileUtils.mkpath(dir) unless File.directory?(dir)
       end
 
+      def ensure_permitted(dir)
```

Comment:
not fond of the name. `ensure_directory_executable_for_all` or `ensure_directory_searchable_for_all` ?